src/components/FolderList.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+interface FolderListProps {
+  folders: string[];
+}
+
+export const FolderList: React.FunctionComponent<FolderListProps> = ({
+  folders
```

Comment:
why would we prefer to have this spread on 3 lines instead of 1 ?

something like:

```typescript
export const FolderList: React.FunctionComponent<FolderListProps> = ( { folders } ) => {
    …
    …
    …
};
```

Reply:
Because this is how prettier is configured or not configured since we use tslint-config-prettier. I used the config from vscode tho, so it might be different once we are truly using tslint-config prettier